charts/customer-center/README.md

```diff
@@ -0,0 +1,158 @@
+| backend.livenessProbe.enabled | bool | `true` | Enable liveness probe on backend |
+| backend.livenessProbe.failureThreshold | int | `6` | Number of tries to perform the probe |
+| backend.livenessProbe.initialDelaySeconds | int | `60` | Number of seconds after the container has started before liveness probe is initiated |
+| backend.livenessProbe.path | string | `"/api/v1/health"` | Path to probe |
+| backend.livenessProbe.periodSeconds | int | `10` | How often (in seconds) to perform the probe |
+| backend.livenessProbe.successThreshold | int | `1` | Minimum consecutive successes for the probe to be considered successful after having failed |
+| backend.livenessProbe.timeoutSeconds | int | `5` | Number of seconds after which the probe times out |
+| backend.readinessProbe.enabled | bool | `false` | Enable readiness probe on backend |
+| backend.readinessProbe.failureThreshold | int | `6` | Number of tries to perform the probe |
+| backend.readinessProbe.initialDelaySeconds | int | `30` | Number of seconds after the container has started before readiness probe is initiated |
+| backend.readinessProbe.path | string | `"/api/v1/health"` | Path to probe |
+| backend.readinessProbe.periodSeconds | int | `10` | How often (in seconds) to perform the probe |
+| backend.readinessProbe.successThreshold | int | `1` | Minimum consecutive successes for the probe to be considered successful after having failed |
+| backend.readinessProbe.timeoutSeconds | int | `5` | Number of seconds after which the probe times out |
+| backend.replicaCount | int | `1` | (int) Number of Backend replicas |
+| backend.resources | object | `{}` | Resource limits for backend |
+| backend.service.externalPort | int | `3000` | External Port of backend service |
+| backend.service.internalPort | int | `3000` | Internal Port of backend service |
+| backend.service.name | string | `"customer-center-backend"` | Backend service name |
+| backend.service.type | string | `"ClusterIP"` | Backend service type |
+| backend.settings.appHost | string | `"my.example.com"` | Application host name |
+| backend.settings.appName | string | `"Customer Center"` | Application name |
+| backend.settings.auth | object | `{"expirePassword":3600,"roleAdmin":"ldap-role-admin","roleEmployee":"ldap-role-employee","secret":"my-super-secret","usersCustomersGroupSearchBase":"cn=groups,ou=customers,cn=accounts,dc=sso,dc=example,dc=com","usersCustomersGroupSearchFilter":"(&(uniqueMember={{dn}})(objectClass=posixGroup))","usersCustomersSearchBase":"ou=customers,cn=accounts,dc=sso,dc=example,dc=com","usersCustomersSearchFilter":"(uid={{username}})","usersGroupSearchBase":"cn=groups,ou=staff,cn=accounts,dc=sso,dc=example,dc=com","usersGroupSearchFilter":"(&(uniqueMember={{dn}})(objectClass=posixGroup))","usersSearchBase":"ou=staff,cn=accounts,dc=sso,dc=example,dc=com","usersSearchFilter":"(uid={{username}})"}` | Authentication settings |
```

Comment:
Please add some docs or doc-annotations to get rid of this huge block of JSON (see values.yaml for a hint on how).

Reply:
The huge JSON block will be removed, thanks for the hint!